sound/soc/sof/intel/hda.h

```diff
@@ -392,6 +392,9 @@ struct sof_intel_hda_dev {
 
 	/* DMIC device */
 	struct platform_device *dmic_dev;
+
+	/* hda codec mask excluding hdmi */
+	u32 hda_codec_mask;
```

Comment:
Is this really needed? Wouldn't it be enough to check codec->jacktbl.used?

Reply:
hda_codec_mask is used by enable & disable WAKEEN, please check other change in the PR. codec->jacktbl.used is not available.